plugins/woocommerce/assets/js/frontend/cart-fragments.js

```diff
@@ -39,7 +39,8 @@ jQuery( function( $ ) {
 		url: wc_cart_fragments_params.wc_ajax_url.toString().replace( '%%endpoint%%', 'get_refreshed_fragments' ),
 		type: 'POST',
 		data: {
-			time: new Date().getTime()
+			time: new Date().getTime(),
+			params: wc_refreshed_fragments_params !== void(0) ? wc_refreshed_fragments_params : null
```

Comment:
I may be missing something, but I'm not sure where this var is defined (or is it a typo)?